biome.json

```diff
@@ -100,6 +99,10 @@
       ".devenv"
     ]
   },
+  "css": {
+    "formatter": { "enabled": false },
+    "linter": { "enabled": false }
+  },
```

Comment:
disabling css linting for now, emotion not supported and linting/formatting our existing css would take a lot of changes